priv/perf/apps/load_test/lib/service/datadog/api.ex

```diff
@@ -0,0 +1,166 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Service.Datadog.API do
+  @moduledoc """
+    Funcions for fetching monitor events from datadog.
+  """
+
+  require Logger
+
+  @datadog_events_api_path "api/v1/events"
+  @datadog_monitor_resolve_path "monitor/bulk_resolve"
+  @datadog_app_url "https://app.datadoghq.com"
+
+  def assert_metrics(environment, start_datetime, end_datetime) do
+    start_unix = DateTime.to_unix(start_datetime)
+    end_unix = DateTime.to_unix(end_datetime)
+
+    # wait for events to be emitted
+    Process.sleep(30_000)
```

Comment:
why do we need to wait? where is 30seconds coming from?

Reply:
monitor events are emitted based on 1 minute time intervals. If events are fetched right after test execution, events may not be published yet

Reply:
Would it be better to poll for this data? Maybe 30s is too much, or too little and we would randomly fail.